transport-native-unix-common/src/main/java/io/netty/channel/unix/Socket.java

```diff
@@ -183,6 +191,10 @@ public final int sendToAddresses(long memoryAddress, int length, InetAddress add
         if (res >= 0) {
             return res;
         }
+
+        if (res == ERROR_ECONNREFUSED_NEGATIVE) {
+            throw new PortUnreachableException("sendto failed");
```

Comment:
(In reference to your previous comment) Sendto is used in other places but it is not used here. Seems confusing to reference an unused native method.